content/en/docs/reference/command-line-tools-reference/feature-gates/disable-node-kube-proxy-version.md

```diff
@@ -9,9 +9,13 @@ stages:
   - stage: alpha
     defaultValue: false
     fromVersion: "1.29"
-    toVersion: "1.30"
+    toVersion: "1.31.0"
   - stage: beta
     defaultValue: true
-    fromVersion: "1.31"
+    fromVersion: "1.31.0"
+    toVersion: "1.31.1"
+  - stage: deprecated
+    defaultValue: false
+    fromVersion: "1.31.1"
```

Comment:
Here's a weird thing.
> The `DisableNodeKubeProxyVersion` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) will be set to `true` in by default in v1.33 and the kubelet will no longer attempt to set the `.status.kubeProxyVersion` field for its associated Node.

Doesn't add up, to me. We don't set deprecated (“you should not use this, we're explicitly asking you not to”) features enabled; if we deprecate something, we're recommending that people don't enable it.

Maybe it's beta and false? Maybe we'll use a different feature gate in the future? But, anyway, it feels really confusing to mark something as deprecated and also commit to locking it enabled.

Reply:
/approve cancel

Reply:
The existence of the gate is to control the process of deprecating something. We *don't* expect people to use it, but it's an escape hatch. Gates in deprecated state that default to true and people can toggle back to false have happened before (SkipReadOnlyValidationGCE, ValidateProxyRedirects, EnableAggregatedDiscoveryTimeout, etc)

Reply:
`SkipReadOnlyValidationGCE` is odd; I wonder why we deprecated it (≅“please don't set this“) _and_ made it true by default. Why didn't we _graduate_ it? 

Have we ever set a gate to deprecated and also announced an intent to mark it true in the future?